packages/core/src/routes/user.ts

```diff
@@ -4,20 +4,13 @@ import { object, string } from 'zod';
 
 import { encryptUserPassword } from '@/lib/user';
 import koaGuard from '@/middleware/koa-guard';
-import { deleteUserById, findAllUsers, findUserById, updateUserById } from '@/queries/user';
+import { deleteUserById, findUserById, updateUserById } from '@/queries/user';
 
 import { AnonymousRouter } from './types';
 
 export default function userRoutes<T extends AnonymousRouter>(router: T) {
-  router.get('/users', async (ctx, next) => {
-    const users = await findAllUsers();
-    ctx.body = users.map((user) => pick(user, ...userInfoSelectFields));
-
-    return next();
-  });
-
   router.get(
-    '/users/:userId',
+    '/user/:userId',
```

Comment:
Let's add an issue for moving management routes to admin router

Reply:
LOG-1448